docker/wait-for/wait-for-objstore.sh

```diff
@@ -4,9 +4,4 @@ echo "** STARTUP - Checking for Object Store connection..."
 
 source /opt/wait-for/wait-for.env
 
-apk add --no-cache mongodb
```

Comment:
How does this work if the mongo client isnt installed?

Reply:
Hi @mdebarros , the wait-for script we are using checks only if the TCP port is opened or not. We are doing the same for kafka. So we don't need the mongodb client here.